contracts/Web3Entry.sol

```diff
@@ -709,13 +722,40 @@ contract Web3Entry is IWeb3Entry, NFTBase, Web3EntryStorage, Initializable, Web3
         emit Events.SetOperator(characterId, operator, block.timestamp);
     }
 
+    function _approveOperatorList(uint256 characterId, address[] calldata operatorList) internal {
+        for (uint256 index = 0; index < operatorList.length; index++) {
+            if (_operatorListByCharacter[characterId][operatorList[index]] == true) {
+                revert Events.alreadyApproved(operatorList[index]);
```

Comment:
If an address is already approved, leaving it as it is is enough. No need to revert.